UI and sounds

apotter3 opened the pull request:
Main Menu basically has everything it needs.
-Added
- Volume settings for music and effects
- A credits and controls page

FeatheredElephant committed:
Merge pull request #9 from FeatheredElephant/main

Merge pull request #7 from FeatheredElephant/UI

FeatheredElephant committed:
Merge pull request #12 from FeatheredElephant/main

Adding character movement

apotter3 committed:
Main menu UI changes

Switched to regular UI building.

Added options panel & music slider.

Transition between options and main menu done

apotter3 committed:
Updated main menu UI & added sounds

Added
- Controls Panel
- Credits Panel
- SFX volume slider
- Sounds on button click
- Downloaded sounds
Deleted
- Main Menu prefab

apotter3 commented:
let me know whatcha think

FeatheredElephant reviewed (changes requested):
Please remove fluff from 3p assets (sample scenes, unused assets, etc.)

apotter3 committed:
Delete unused files

Deleted unused sounds.

Left some sounds that could be used for other use.

Created sounds folder

apotter3 committed:
Deleted Unity TMP fluff

apotter3 commented:
-Deleted
- Unused TMP files
- Unused sounds

Left some sounds I think would be good for some effects going forward (Slashing, enemy sounds)

Sample scenes have been removed in most recent commits
